server/routes/api/v1/surrenderRouter.js

```diff
@@ -0,0 +1,19 @@
+import express from "express"
+
+import SurrenderApplication from "../../../models/SurrenderApplication.js"
+
+const surrenderRouter = new express.Router()
+
+surrenderRouter.post("/", async (req, res) => {
+  try {
+    const newSurrender = new SurrenderApplication(req.body)
+    const petId = await newPet.save()

```

Comment:
where is `newPet` coming from here? This looks like a bug?

Reply:
This was on purpose.  The save method for pet returns the petId which is then fed to the save method for the surrender application so the pet primary id and surrender application pet foreign key are the same and reference each other